### PR TITLE
rpc Incorrect location of the Back button [sc-16343]

### PR DESCRIPTION
**Pull request by vitaliipustovyi**

### CODE REVIEW

File: packages/ui/src/organisms/authentication/other-sign-options.tsx/index.tsx [hunk 1]
```diff
@@ -41,10 +43,15 @@ export const AuthOtherSignOptions = ({
...
-        className="absolute w-6 transition-opacity cursor-pointer hover:opacity-50 shrink-0"
+        className="absolute w-6 transition-opacity cursor-pointer hover:opacity-50 shrink-0 top-5 left-5"
         onClick={onBack}
       />
-      <AuthAppMeta applicationURL={appMeta} />
+      <AuthAppMeta
+        applicationURL={appMeta}
+        withLogo={!isIdentityKit}
+        title={isIdentityKit ? "Sign in" : undefined}
+        subTitle={isIdentityKit ? "to continue to" : undefined}
```
Comment (olekid) [inline packages/ui/src/organisms/authentication/other-sign-options.tsx/index.tsx:53]: lets pass prop like title and subTitle, not isIdentitykit

Comment (vitaliipustovyi) [reply packages/ui/src/organisms/authentication/other-sign-options.tsx/index.tsx:53]: here are 3 values depend on identitykit: withLogo, title, subtitle.
Do you mean to pass these values as props from coordinator ?

Comment (olekid) [reply packages/ui/src/organisms/authentication/other-sign-options.tsx/index.tsx:53]: exactly

Comment (vitaliipustovyi) [reply packages/ui/src/organisms/authentication/other-sign-options.tsx/index.tsx]: DOne

**Pull request merged**